packages/portalnetwork/src/client/client.ts

```diff
@@ -1130,7 +1151,8 @@ export class PortalNetwork extends (EventEmitter as { new (): PortalNetworkEvent
     }
   }
 
-  private sendPortalNetworkResponse = (
+  private sendPortalNetworkResponse = async (
+    src: INodeAddress,
```

Comment:
can we rename either `src` or `srcId`?  just being nit-picky but the similarity bugs me.
also, this refers the receiving node's info, right?  feels weird to me that what gets passed as `src` and `srcId` end up creating `dstPeerId` and `dstAddr`

Reply:
I've removed `scrId` from all of these function signatures since the id value is part of `src`.